src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs

```diff
@@ -162,6 +192,61 @@ public class Ado : NotificationsBase, IAdo {
             });
         }
 
+        // Validate AreaPath exists
+        try {
+            if (config.AdoFields.TryGetValue("System.AreaPath", out var areaPathString)) {
+                var validateAreaPath = await ValidatePath(config.Project, areaPathString, TreeStructureGroup.Areas, witClient);
+                if (!validateAreaPath.IsOk) {
+                    return validateAreaPath;
+                }
+            }
+        } catch (VssUnauthorizedException e) {
+            return OneFuzzResultVoid.Error(ErrorCode.ADO_VALIDATION_INVALID_PAT, new string[] {
+                "The provided PAT may be missing scopes. We were able to connect with it but unable to validate the fields.",
```

Comment:
Do we need additional scopes on a PAT (other than work item list/create) to use this 'get classification nodes' endpoint?

Reply:
[Nope](https://learn.microsoft.com/en-us/rest/api/azure/devops/wit/classification-nodes/get-classification-nodes?view=azure-devops-rest-7.1&tabs=HTTP), good point